internal/nomad/nomad.go

```diff
@@ -359,12 +356,11 @@ func handleAllocationEvent(startTime int64, allocations storage.Storage[*allocat
 // This allows the handling of startups and re-placements of allocations.
 func handlePendingAllocationEvent(alloc *nomadApi.Allocation,
 	allocations storage.Storage[*allocationData], callbacks *AllocationProcessoring) {
-	log.WithField("alloc_id", alloc.ID).Debug("Handle Pending Allocation Event")
 	if alloc.DesiredStatus == structs.AllocDesiredStatusRun {
 		allocData, ok := allocations.Get(alloc.ID)
 		if ok && allocData.allocClientStatus != structs.AllocClientStatusRunning {
 			// Pending Allocation is already stored.
-			log.WithField("alloc_id", alloc.ID).Debug("Pending Allocation already stored")
+			log.WithField("alloc_id", alloc.ID).Trace("Pending Allocation already stored")
```

Comment:
Do you expect that (normally) we see this Trace message being logged? Just by reading the code and your explanation, I would argue that we first process the `PlanResult` and then the `AllocationUpdated` (thus causing this message)

Reply:
Yes, I would agree that this log would be triggered for almost every started allocation. In order to not blur the Poseidon log, I degraded it to the `trace` level that is commonly not activated. But, I can also remove it?!

Reply:
At least, I think the message could be misleading, because it somewhat implies that the processing an already pending allocation is somewhat unexpected. In case we keep it (see above), I would rather opt to remove the line and expand the comment above (stating that we get two events per pending allocation and -- by design -- are just interested in a set of pending allocation, implying to remove duplicated columns).